pyproject.toml

```diff
@@ -199,6 +199,8 @@ update-check = false
 target-version = "py38"
 
 select = [
+  # implicit string concatenation
+  "ISC"
```

Comment:
needs a comma at the end of the line - did you try running this?

Reply:
Hello when i am running the **pre-commit run ruff --all-files** . its shows this output. Can u help me with running please.
![Capture](https://user-images.githubusercontent.com/73106537/220363388-84023ab2-dded-4927-91d2-88976fbbd801.JPG)

Reply:
please check https://pandas.pydata.org/docs/dev/development/contributing_environment.html